cli/inspect/inspector.go

```diff
@@ -76,23 +77,32 @@ func Inspect(out io.Writer, ref string, tmplStr string, getRef GetRefFunc) error
 	return nil
 }
 
-// RunInspectFunc is a function which used by MultiInspect
-type RunInspectFunc func(args []string) error
+// combineErrors is a function which used by MultiInspect to merge multiple errors
+// into one error.
+func combineErrors(errs []error) error {
```

Comment:
Could we encapsulate this function into package pkg or something like that?
Because I think this function is common enough.

Reply:
Sure.I agree with what you say.I will move it and add a unit test.

Reply:
@allencloud Done.
And could you help to review this pr again? Thanks a lot.